lib/tree-view.coffee

```diff
@@ -96,11 +96,16 @@ class TreeView extends View
     @on 'mousedown', '.entry', (e) =>
       @onMouseDown(e)
     @on 'mousedown', '.tree-view-resize-handle', (e) => @resizeStarted(e)
-    @on 'dragstart', '.entry', (e) => @onDragStart(e)
-    @on 'dragenter', '.entry.directory > .header', (e) => @onDragEnter(e)
-    @on 'dragleave', '.entry.directory > .header', (e) => @onDragLeave(e)
-    @on 'dragover', '.entry', (e) => @onDragOver(e)
-    @on 'drop', '.entry', (e) => @onDrop(e)
+    @on 'dragstart', '.entry', (e) =>
+        @onDragStart(e)
+    @on 'dragenter', '.entry.directory > .header', (e) =>
+        @onDragEnter(e)
+    @on 'dragleave', '.entry.directory > .header', (e) =>
+        @onDragLeave(e)
+    @on 'dragover', '.entry', (e) =>
+        @onDragOver(e)
+    @on 'drop', '.entry', (e) =>
+        @onDrop(e)
```

Comment:
Yeah, I had a hack in there actually that was stupid and I removed it.

What I really needed to do was refresh that file, but I didn't.  I will do that next time.